ext/windows/facter.bat

```diff
@@ -0,0 +1,8 @@
+@echo off
+SETLOCAL
+if exist "%~dp0environment.bat" (
+  call "%~dp0environment.bat" %0 %*
+) else (
+  SET PATH=%~dp0;%PATH%
```

Comment:
I was almost right, using `SET PATH="%~dp0;%PATH%"` creates an invalid PATH. It needs to be `SET "PATH=%~dp0;%PATH%"`.